readthedocs/projects/views/public.py

```diff
@@ -177,10 +177,13 @@ def get(self, request, project_slug, *args, **kwargs):
             ).first()
 
         if version:
-            last_build = version.builds.filter(
-                type='html',
-                state='finished',
-            ).order_by('-date').first()
+            last_build = (
+                version.builds
+                .filter(type='html', state='finished')
```

Comment:
I feel like we have another bug where builds are default set to `finished` when they are created. We should probably change that as well so we don't hit this as well 👍

Reply:
Do you mean this is another problem (not something I need to change here)?

Reply:
Yea, it can be done in another PR.